### PR TITLE
Update Translation pt_BR

### PR DESCRIPTION
**Pull request by gleissonjoaquim3**
Update of the Brazilian Portuguese translation of the file pt_BR.po. Several improvements have been made that have not yet been translated. So I hope that more fans can use this app Fantastic :)

### CODE REVIEW

File: po/extra/pt_BR.po [hunk 1]
```diff
@@ -8,399 +8,440 @@ msgstr ""
...
 
 #: data/com.github.alainm23.planner.desktop.in:8
 msgid "com.github.alainm23.planner"
-msgstr ""
+msgstr "com.github.alainm23.planner"
 
 #: data/com.github.alainm23.planner.desktop.in:12
 msgid "development;task;tasks;project;todo;event;events;calendar;todoist;"
 msgstr ""
+"desenvolvimento; tarefa; tarefas; projeto; todo; evento; eventos; "
```
Comment (alainm23) [inline po/extra/pt_BR.po]: Do not separate the elements with space.

File: po/extra/pt_BR.po [hunk 2]
```diff
@@ -8,399 +8,440 @@ msgstr ""
...
 #: data/com.github.alainm23.planner.appdata.xml.in:10
 msgid "Everyone needs a task manager and Planner adapts to anyone's needs"
 msgstr ""
+"Todo mundo precisa de um gerenciador de tarefas e o Planejador se adapta a "
+"todas as necessidades"
 
 #: data/com.github.alainm23.planner.appdata.xml.in:12
 msgid "🚀️ Planner 2 is here..."
-msgstr ""
+msgstr "O Planejador 2 está aqui..."
```
Comment (alainm23) [inline po/extra/pt_BR.po]: Please use the emoticon.

**Comment by gleissonjoaquim3** [comment]
@alainm23 I used the updated fork,  no conflitcts with base branch, but fail... :(

File: po/extra/pt_BR.po [hunk 3]
```diff
@@ -8,399 +8,440 @@ msgstr ""
...
 "4. Be even more organized - Add a duedate to your tasks, create labels, use "
 "checklists."
 msgstr ""
+"4. Seja ainda mais organizado - Adicione uma data de vencimento às suas "
+"tarefas, crie etiquetas, use listas de verificação."
 
 #: data/com.github.alainm23.planner.appdata.xml.in:32
 msgid "☁️ Support for Todoist:"
-msgstr ""
+msgstr "☁️Suporte ao Todoist:"
```
Comment (alainm23) [inline po/extra/pt_BR.po]: Add a small space after the emoticon.

**Review by alainm23 — CHANGES_REQUESTED** [review]
There seems to be a problem with Travis, I'll have to check this manually.

It's just a few details, man. Everything else is fine.

File: po/extra/pt_BR.po (hunk 1 above)
Comment (gleissonjoaquim3) [reply po/extra/pt_BR.po]: Solved.

File: po/extra/pt_BR.po (hunk 2 above)
Comment (gleissonjoaquim3) [reply po/extra/pt_BR.po]: Solved.

File: po/extra/pt_BR.po (hunk 3 above)
Comment (gleissonjoaquim3) [reply po/extra/pt_BR.po]: Solved

**Review by gleissonjoaquim3 — COMMENTED** [review]
I completely revised the 2 files, I apologize for some errors.

**Comment by alainm23** [comment]
Thank you very much, everything is fine :)

**Pull request merged**